aptos-move/aptos-workspace-server/src/lib.rs

```diff
@@ -181,3 +197,19 @@ pub async fn run_all_services() -> Result<()> {
 
     Ok(())
 }
+
+#[derive(Parser)]
+pub enum WorkspaceCommand {
+    Run {
+        #[arg(long, default_value_t = 1800)]
+        timeout: u64,
```

Comment:
A comment and maybe better naming could be nice here. My first thought was that this would be a startup or shutdown timeout, but it really seems like a "max lifespan".

I don't fully know exactly how you use workspace, but I suppose there is a good reason that we always do this, vs making it optional?

Reply:
Also it'd be good to suffix whatever name we choose with `_secs`